DeploymentExperiences.md

```diff
@@ -1,8 +1,16 @@
-- Summarize the key steps in the deployment process. 
+- Summarize the key steps in the deployment process.
   - What went well?
-  - What challenges did you face? 
+  - What challenges did you face?
   - How far did you get?
 - We have utilized docs from several different projects over the course of this Sprint.
-  - Which docs did you think were the most helpful? 
-  - Name one or two specific things about this resource that you think sets it apart from other docs you have referenced in the past. 
+  - Which docs did you think were the most helpful?
+  - Name one or two specific things about this resource that you think sets it apart from other docs you have referenced in the past.
 
+
+Deploying to Heroku requires a few key steps. First, one has to make an account with Heroku. This is as straightforward as signing up for any other service online. Then, from the command line, set a few key variables (these can also be reached online from the Heroku dashboard). Finally, the app has to be deployed by choosing a unique app name and pushing your repo to Heroku's git.
```

Comment:
Great job outlining your steps for your deployment process, very well explained.